event-adapter/calls.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
```

Comment:
I think you should exclude this file, I had it in my repo initially for testing purposes, but I don't think it should stay there.